bindings/py/cpp_src/bindings/encoders/py_ScalarEncoder.cpp

```diff
@@ -0,0 +1,54 @@
+/* ----------------------------------------------------------------------
+ * Numenta Platform for Intelligent Computing (NuPIC)
+ * Copyright (C) 2019, David McDougall
+ * The following terms and conditions apply:
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses.
+ *
+ * http://numenta.org/licenses/
+ * ---------------------------------------------------------------------- */
+
+#include <bindings/suppress_register.hpp>  //include before pybind11.h
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl.h>
+namespace py = pybind11;
+
+#include <nupic/encoders/ScalarEncoder.hpp>
+
+using namespace nupic;
+
+namespace nupic_ext
```

Comment:
put using ::nupic under this ns

Reply:
I don't understand what you're asking.  I don't recognise that syntax.  Can you teach me or point me to a reference explaining it?

Reply:
sorry for the shorthand, I just wanted to 

```
namespace nupic_ext {
using namespace nupic; 
...
```

the ::nupic refers to a namespace nupic;